packages/@sanity/portable-text-editor/src/utils/values.ts

```diff
@@ -50,12 +57,20 @@ export function toSlateValue(
         })
         if (!hasInlines && Element.isElement(block)) {
           // Original object
-          return block as Node
+          return block
         }
         return keepObjectEquality({_type, _key, ...rest, children}, keyMap)
       }
-      return keepObjectEquality({_type, _key, children: [{text: ''}], value: rest}, keyMap)
-    }) as Node[]
+      return keepObjectEquality(
+        {
+          _type,
+          _key,
+          children: voidChildren,
+          value: rest,
+        },
+        keyMap
+      )
+    }) as Descendant[]
```

Comment:
It this type casting required? `as Descendant[]`

Reply:
Yes, here it actually is, because it's a util to convert values.